yahtzee.py

```diff
@@ -197,28 +246,44 @@ def __init__(self):
             
         print('You rolled: ', player1.current_throw)
```

Comment:
This loop is very clever and it's nice that you implemented the option to throw again